support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -358,8 +358,12 @@ footer[role="contentinfo"] a {
 
 .gu-content__content--flex {
   display: flex;
-  justify-content: space-between;
+  justify-content: space-around;
   flex-direction: row-reverse;
+
+  @include mq($from: leftCol) {
+    justify-content: flex-end;
```

Comment:
Just checking, have you checked how this looks at the 320px (small mobile) and 375px (mobile) breakpoints?

Reply:
yes